var/da/da_radiance/da_read_obs_netcdf4ahi_jaxa.inc

```diff
@@ -319,6 +349,28 @@ subroutine da_read_obs_netcdf4ahi_jaxa (iv, infile_tb, infile_tbp)
          'solar azimuth angle(pixel=1,scan=1): ',sol_azi(1,1)		 		  
    ! close infile_tb file 		 
       iret = nf90_close(ncid)
+
+      !  Read the level-2 cloud information file from JAXA
+      !  This must be on the same grid as everything else
+      allocate(cloud_mask(nlongitude,nlatitude))
+      cloud_mask = -9999.0
+      inquire(file='L2AHICLP', exist=fexist)
+      if ( fexist ) then
+         iret = nf90_open('L2AHICLP', nf90_NOWRITE, ncid)
```

Comment:
If the Level-2 file does not exist, is the whole code still work? I guess so, but want to make sure. Level-2 data read should be optional.

Reply:
Yes, it will work if the L-2 file is missing.  If the file isn't there, `cloud_mask = -9999.0 `everywhere. The diagnostics will have this useless data, but it won't break WRFDA or impact any other fields/variables.